InstallingForge.rst

```diff
@@ -1,7 +1,7 @@
 Installing Forge
 ================
 
-Go to the `Forge download page <https://simplifier.net/forge>`__ to
+Go to the `Forge download page <https://simplifier.net/downloads/forge>`__ to
```

Comment:
@Rob5045 I wouldn't make this change. The difference between the two pages is that /forge is accessible for not-logged-in users, /downloads/forge is not. So it's a bit clearer why you should make an account.

Reply:
OK reverted link to previous url.